src/util/constants.ts

```diff
@@ -20,127 +20,157 @@ export const baseEndpoints = {
+  WEBHOOK_GITHUB: (webhookID: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/webhooks/${webhookID}/${token}/github`,
 
   // Application Endpoints
   COMMANDS: (botID: string) =>
     `${baseEndpoints.BASE_URL}/applications/${botID}/commands`,
-  COMMANDS_GUILD: (botID: string, id: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${id}/commands`,
-  COMMANDS_ID: (botID: string, id: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/commands/${id}`,
-  COMMANDS_GUILD_ID: (botID: string, id: string, guildID: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands/${id}`,
+  COMMANDS_GUILD: (botID: string, guildID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands`,
+  COMMANDS_ID: (botID: string, commandID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/commands/${commandID}`,
+  COMMANDS_GUILD_ID: (botID: string, commandID: string, guildID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands/${commandID}`,
 
   // Interaction Endpoints
-  INTERACTION_ID_TOKEN: (id: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/interactions/${id}/${token}/callback`,
-  INTERACTION_ORIGINAL_ID_TOKEN: (id: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${id}/${token}/messages/@original`,
+  INTERACTION_ID_TOKEN: (interactionID: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/interactions/${interactionID}/${token}/callback`,
+  INTERACTION_ORIGINAL_ID_TOKEN: (interactionID: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/webhooks/${interactionID}/${token}/messages/@original`,
   INTERACTION_ID_TOKEN_MESSAGEID: (
-    id: string,
+    applicationID: string,
     token: string,
     messageID: string,
   ) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${id}/${token}/messages/${messageID}`,
+    `${baseEndpoints.BASE_URL}/webhooks/${applicationID}/${token}/messages/${messageID}`,
 
   // User endpoints
-  USER: (id: string) => `${baseEndpoints.BASE_URL}/users/${id}`,
+  USER: (userID: string) => `${baseEndpoints.BASE_URL}/users/${userID}`,
   USER_BOT: `${baseEndpoints.BASE_URL}/users/@me`,
-  USER_AVATAR: (id: string, icon: string) =>
-    `${baseEndpoints.CDN_URL}/avatars/${id}/${icon}`,
+  USER_GUILDS: `${baseEndpoints.BASE_URL}/@me/guilds`,
+  USER_AVATAR: (userID: string, icon: string) =>
+    `${baseEndpoints.CDN_URL}/avatars/${userID}/${icon}`,
   USER_DEFAULT_AVATAR: (icon: number) =>
     `${baseEndpoints.CDN_URL}/embed/avatars/${icon}.png`,
-  USER_CREATE_DM: `${baseEndpoints.BASE_URL}/users/@me/channels`,
+  USER_DM: `${baseEndpoints.BASE_URL}/users/@me/channels`,
+  USER_CONNECTIONS: `${baseEndpoints.BASE_URL}/users/@me/connections`,
+  USER_NICK: (guildID: string) => `${GUILDS_BASE(guildID)}/members/@me/nick`,
+
+  // Voice endpoints
+  VOICE_REGIONS: `${baseEndpoints.BASE_URL}/voice/regions`,
```

Comment:
Are we using this endpoint anywhere? I don't think this is needed if we are not using it; in case we do use it in the future, we can always add it.

Reply:
Basically not. But if you want to create a bot which has a interactive region selection tool when creating servers this is nice to have.

Reply:
Minimalism, one of the most important goals of Discordeno. In case users do, they can always define them by themselves.

Reply:
Hmm, I thought 100% API coverage is a nice plus point.

Reply:
Feel free to add it back when you want to add corresponding functionality.